platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/StateConfig.java

```diff
@@ -98,7 +95,6 @@ public record StateConfig(
         @ConfigProperty(defaultValue = "false") boolean stateHistoryEnabled,
         @ConfigProperty(defaultValue = "false") boolean debugStackTracesEnabled,
         @ConfigProperty(defaultValue = "emergencyRecovery.yaml") String emergencyStateFileName,
-        @ConfigProperty(defaultValue = "1") int signedStateFreq,
```

Comment:
why remove this config?

Reply:
We discussed this and came to consensus to remove the config